### PR TITLE
Fix announcements and warning for not plugging in the usb

### PR DESCRIPTION
**Pull request by LiCody**
### Description

fixes crashing due to display being temporarily removed and the disable announcement task failing if announcements are deleted

### Testing Done

Tested on robot

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

### CODE REVIEW

File: src/software/jetson_nano/ansible/playbooks/deploy_nano.yml [hunk 1]
```diff
@@ -71,6 +71,7 @@
           register: result
           args:
             chdir: ~/thunderbots_binaries
+          ignore_errors: yes
```
Comment (nimazareian) [inline src/software/jetson_nano/ansible/playbooks/deploy_nano.yml]: Do we not want to add `when: "'stop' in actions"`?

**Review by nimazareian — COMMENTED** [review]
So does this not show any more errors when normally flashing? Since I feel like I had some red errors pop up even with `ignore_errors: yes`, though Thunderloop was still flashed fine.

**Comment by LiCody** [comment]
> So does this not show any more errors when normally flashing? Since I feel like I had some red errors pop up even with `ignore_errors: yes`, though Thunderloop was still flashed fine.

Yes u still get red errors with `ignore_errors: yes` they wont stop the script from finishing. I disabled the red errors for stopping announcements so that task will always pass.

File: src/software/jetson_nano/ansible/playbooks/deploy_nano.yml (hunk 1 above)
Comment (LiCody) [reply src/software/jetson_nano/ansible/playbooks/deploy_nano.yml]: added

**Review by nimazareian — APPROVED** [review]

File: src/software/jetson_nano/services/power.cpp [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
 #include <cstdint>
 
 #include "proto/power_frame_msg.nanopb.h"
 
 PowerService::PowerService()
 {
+    if (!boost::filesystem::exists(DEVICE_SERIAL_PORT))
+    {
+        LOG(FATAL) << "PLUG THE USB INTO THE JETSON NANO";
+    }
```
Comment (itsarune) [inline src/software/jetson_nano/services/power.cpp:14]: will we not have comm over UART through the jetson pin headers?

**Review by itsarune — APPROVED** [review]
🔥

File: src/software/jetson_nano/services/power.cpp (hunk 2 above)
Comment (LiCody) [reply src/software/jetson_nano/services/power.cpp:14]: this will get removed when we switch to UART instead of USB

**Pull request merged**